copy-files-pipeline/copy-files.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+set -e
+# This is a simple example to copy files from any source environment to target CDE.
+# Note: This script doesn't handle the token expiry or any API exceptions/errors.
+
+SOURCE_ENV_NAME='test'
+
+# Get Cloud Authentication token. For more details: https://docs.acquia.com/acquia-cloud/develop/api/auth/
+TOKEN=$(curl -sS -X POST -u "${CLOUD_API_KEY}:${CLOUD_API_SECRET}" -d "grant_type=client_credentials" https://accounts.acquia.com/api/auth/oauth/token | python -c "import sys, json; print json.load(sys.stdin)['access_token']")
```

Comment:
I love one-liners like this one, but you're not checking after if TOKEN was empty.
Because of credentials, because of the python dependency, because of an unexpected object returned...